Content.Server/StationEvents/Events/IonStormRule.cs

```diff
@@ -1,64 +1,15 @@
 using System.Linq;
 using Content.Server.Silicons.Laws;
```

Comment:
```suggestion
```

Reply:
I don't understand this change, it doesn't build if 'using Content.Server.Silicons.Laws;' is removed.

Reply:
Oops, my bad. VsCodium sometimes lies to me about unnecessary imports.